src/submission-service.ts

```diff
@@ -128,11 +128,14 @@ async function processPendingQueue(): Promise<void> {
 async function submit({
   formSubmission,
   paymentReceiptUrl,
-  schedulingReceiptUrl,
+  schedulingUrlConfiguration,
```

Comment:
Docs this need to change as well.

Reply:
This is currently not in any docs... I suppose I'll add them

Reply:
nor are half the functions in the scheduling service

Reply:
I only documented `handleSchedulingQuerystring()` as the other functions in the service should not be used externally.

Reply:
We could move them to another file, if we don't want them exported on the `schedulingService`

Reply:
Yeh we should do that, if they are not meant for use